.travis.yml

```diff
@@ -1,7 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
```

Comment:
Ok, nevermind, I checked if Travis docs were outdated, then I reverted the change in `.travis.yml` and the build succeeded; however, users must be aware that this branch fails with Node < 4.5.x (I tested it with 4.4.x and it fails).